applications/GeoMechanicsApplication/custom_conditions/axisymmetric_line_normal_fluid_flux_2D_diff_order_condition.cpp

```diff
@@ -41,8 +41,13 @@ AxisymmetricLineNormalFluidFlux2DDiffOrderCondition::AxisymmetricLineNormalFluid
 Condition::Pointer AxisymmetricLineNormalFluidFlux2DDiffOrderCondition::Create(
     IndexType NewId, NodesArrayType const& ThisNodes, PropertiesType::Pointer pProperties) const
 {
-    return Condition::Pointer(new AxisymmetricLineNormalFluidFlux2DDiffOrderCondition(
-        NewId, GetGeometry().Create(ThisNodes), pProperties));
+    return Create(NewId, GetGeometry().Create(ThisNodes), pProperties);
```

Comment:
In the u_pw_condition it uses this->Create. Why differently here.

Reply:
`UPwCondition` is a class template, contrary to `AxisymmetricLineNormalFluidFlux2DDiffOrderCondition`, which is a non-template class. Strictly speaking, adding `this->` is only required when a member function of a class template references a base member. See Section 26.3.7. of the C++ Programming Language book (4th edition). To make it a little less complex, I usually use `this->` in template code when referencing any member (whether it is a base member or not). For non-template classes, I don't add the `this->` qualification. Hope this helps...?

Reply:
Yes. Thank you.